packages/manager/modules/billing-components/src/components/commitment-duration/commitment-duration.component.js

```diff
@@ -3,6 +3,7 @@ import template from './commitment-duration.html';
 
 export default {
   bindings: {
+    isToPreselectDefaultDuration: '<?',
```

Comment:
Component name already mention `duration` so
```suggestion
    preselectDefault: '<?',
```